86 implement comparison sheets

markushal opened the pull request.

joAschauer reviewed (changes requested):
the option "10 cheapest" does not make sense for "Costs by data scenario" and "Costs by renewable electricity source". We could think about removing this. Otherwise looks good!

markushal commented:
> the option "10 cheapest" does not make sense for "Costs by data scenario" and "Costs by renewable electricity source". We could think about removing this. Otherwise looks good!

You are so right. I fixed this, and also added the option to highlight the current selection in the bar plots. Fine like this?

markushal commented:
@joAschauer : You approved the PR yesterday. I added three new commit today and was able to merge the complete PR just now, including the three new commits. Isnt it strange that you did not need to review them?
